Content.Server/_CD/Traits/LayEmoteComponent.cs

```diff
@@ -0,0 +1,21 @@
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server._CD.Traits;
+
+/// <summary>
+/// Set players' blood to coolant, and is used to notify them of ion storms
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(LayEmoteSystem))]
+public sealed partial class LayEmoteComponent : Component
+{
+    [DataField("layEmote", customTypeSerializer: typeof(PrototypeIdSerializer<EmotePrototype>))]
+    public string LayEmoteId = "LayDown";
+
+    [DataField("standEmote", customTypeSerializer: typeof(PrototypeIdSerializer<EmotePrototype>))]
+    public string StandEmoteId = "StandUp";
```

Comment:
Shouldn't these be `ProtoId`?

Reply:
Most uses I can find upstream use string